packages/interim/src/renovation/component_wrapper/common/template_wrapper.ts

```diff
@@ -2,86 +2,149 @@ import { InfernoComponent, InfernoEffect } from '@devextreme/runtime/inferno';
 // eslint-disable-next-line spellcheck/spell-checker
 import { findDOMfromVNode } from 'inferno';
 import { shallowEquals } from '../../utils/shallow_equals';
-import { replaceWith } from '../../../core/utils/dom';
-import $ from '../../../core/renderer';
+// eslint-disable-next-line import/named
+import $, { dxElementWrapper } from '../../../core/renderer';
 import domAdapter from '../../../core/dom_adapter';
-import { getPublicElement } from '../../../core/element';
-import { removeDifferentElements } from '../utils/utils';
```

Comment:
Looks like we can delete removeDifferentElements from sourcies

Reply:
done